### PR TITLE
add DNI generator to spanis person

### PR DESCRIPTION
**Pull request by mikk150**
I needed to generate DNI on unit tests and I use faker as data provider. I couldn't find DNI anywhere, so I implemented it

### CODE REVIEW

File: src/Faker/Provider/es_ES/Person.php [hunk 1]
```diff
@@ -68,4 +70,12 @@ public static function suffix()
     {
         return static::randomElement(static::$suffix);
     }
+
+    public static function dni()
```
Comment (fzaninotto) [inline src/Faker/Provider/es_ES/Person.php]: please add a link to the dni specification

File: src/Faker/Provider/es_ES/Person.php [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 class Person extends \Faker\Provider\Person
 {
+    private static $letters=array('T', 'R', 'W', 'A', 'G', 'M', 'Y', 'F', 'P', 'D', 'X', 'B', 'N', 'J', 'Z', 'S', 'Q', 'V', 'H', 'L', 'C', 'K', 'E', 'T');
```
Comment (fzaninotto) [inline src/Faker/Provider/es_ES/Person.php]: `letters` is a too specific name. Please rename it to make it obvious that this relates to DNI validation

**Comment by fzaninotto** [comment]
You should add a unit test for this new formatter.

File: src/Faker/Provider/es_ES/Person.php (hunk 1 above)
Comment (mikk150) [inline src/Faker/Provider/es_ES/Person.php]: https://es.wikibooks.org/wiki/Algoritmo_para_obtener_la_letra_del_NIF#Algoritmo

File: src/Faker/Provider/es_ES/Person.php (hunk 2 above)
Comment (mikk150) [inline src/Faker/Provider/es_ES/Person.php]: maybe crcMap?

Comment (mikk150) [inline src/Faker/Provider/es_ES/Person.php]: @fzaninotto is it better now?

File: src/Faker/Provider/es_ES/Person.php (hunk 1 above)
Comment (fzaninotto) [inline src/Faker/Provider/es_ES/Person.php]: Please add it as a `@link` annotation in the method phpDoc

**Pull request merged**